src/scripts/tock-ops-report.js

```diff
@@ -4,13 +4,13 @@ const scheduler = require("node-schedule");
 const {
   dates: { getCurrentWorkWeek },
   slack: { postMessage },
-  tock: { get18FTockTruants },
+  tock: { get18FUsersWhoHaveNotTocked },
 } = require("../utils");
 
 module.exports = (app, config = process.env) => {
   if (!config.TOCK_API || !config.TOCK_TOKEN) {
     app.logger.warn(
-      "Tock truant report disabled: Tock API URL or access token is not set"
+      "Tock compliance report disabled: Tock API URL or access token is not set"
```

Comment:
Oooh, good point. I almost want to go somewhere silly like "Naughty Tockers" or something, but that could easily sound punitive rather than silly. Given how this is used, I'm going to leave it for now. It should pretty much never show up anywhere.